packages/vendure-plugin-invoices/src/strategies/s3-storage.strategy.ts

```diff
@@ -15,8 +14,10 @@ export interface Config {
   expiresInSeconds: number;
   bucket: string;
   region?: string;
+  // eslint-disable-next-line  @typescript-eslint/no-explicit-any
```

Comment:
Maybe we should disable some rules like this one that occur a lot. Like the usage of any and unsafe member access for Error logs. We disabled them everywhere, so maybe its better to relax our eslint config a bit.

WDYT?